kedro/framework/hooks/specs.py

```diff
@@ -135,6 +135,7 @@ def on_node_error(
 class PipelineSpecs:
     """Namespace that defines all specifications for a pipeline's lifecycle hooks."""
 
+    # pylint: disable=missing-param-doc
```

Comment:
Had to disable this check for this specific class. It's complaining about missing parameters in doc string for `run_params` in all the below hook_specs despite it being mentioned in all of them. If you remove the tab this check passes, but then sphinx linkcheck is unhappy and raises `sphinx.errors.SphinxWarning: /home/circleci/project/kedro/framework/hooks/specs.py:docstring of kedro.framework.hooks.specs.PipelineSpecs.after_pipeline_run:20:Definition list ends without a blank line; unexpected unindent.
`